apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -218,7 +218,14 @@ export default function ListBox({
   }
 
   const isVertical = listLayout !== 'horizontal';
-  const count = layout.qListObject.qSize.qcy;
+
+  let count = layout.qListObject.qSize.qcy;
+
+  // Should not allow a count of zero since it will stop executing loadMoreItems
+  // this could happen if a seach returns an empty result
+  if (count === 0) {
```

Comment:
I tested this and it seems to work as expected and solve the problem!
Would be nice with a unit test covering this though.

Reply:
https://github.com/qlik-oss/nebula.js/pull/886